src/main/java/io/fabric8/jenkins/openshiftsync/BuildConfigToJobMapper.java

```diff
@@ -106,6 +98,31 @@ public static FlowDefinition mapBuildConfigToFlow(BuildConfig bc) throws IOExcep
 		}
 	}
 
+	public static GitSCM getGitSCM(BuildSource source, BuildConfig bc){
```

Comment:
does it require to be public? and static ?

Reply:
I ahve changed it to private, it needs to be static though, as we are referencing it from a static context,